### PR TITLE
doc / ongoing v0.37

### PR DESCRIPTION
**Pull request by JeremyKono**
Ongoing docs and release notes for v0.37

### CODE REVIEW

File: content/release-notes/0.37.0.mdx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+The idea of this strategy is to open a position in one direction and execute an order in the opposite direction on the spot. Learn more about it here #[2977](https://github.com/CoinAlpha/hummingbot/pull/2977).
+
+## Improvement to Liquidity Mining strategy
+
+In v0.36, we introduced the Liquidity Mining strategy. In v0.37, we made improvements to give users a better experience when using this strategy, we added two new parameters [`max_spread`](/strategies/liquidity-mining/#max_spread) and [`max_order_age`](/strategies/liquidity-mining/#max_order_age). Tables are also added to let you see more detail about your base and quote balance and also the campaign. Another improvement is better balance distribution on the selected `token`. Learn more about it here #[2961](https://github.com/CoinAlpha/hummingbot/pull/2961).
```
Comment (dennisocana) [inline content/release-notes/0.37.0.mdx:44]:
```suggestion
We added two new parameters [`max_spread`](/strategies/liquidity-mining/#max_spread) and [`max_order_age`](/strategies/liquidity-mining/#max_order_age) to Liquidity Mining strategy to give users a more functionality when using the strategy.

Tables are added to display the campaign, base and quote balance in more detail. Balance distribution has also been improved on the user's selected `token`.

Learn more about it here #[2961](https://github.com/CoinAlpha/hummingbot/pull/2961).
```

Comment (dennisocana) [reply content/release-notes/0.37.0.mdx:44]: "~~Learn more about it here #[2961](https://github.com/CoinAlpha/hummingbot/pull/2961)~~" ditto

File: content/release-notes/0.37.0.mdx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+Perpetual protocol is a decentralized protocol to trade perpetual contracts for every asset powered by a Virtual Automated Market Maker (vAMM). The project aims to create novel financial instruments by democratizing Futures and other crypto-asset derivatives.
+
+Read more about how to use the Perpetual Finance connector [here](/protocol-connectors/perp-fi)
+
+## Introducing the new Spot Perpetual Arbitrage strategy (Beta)
+
+In v0.37, we introduce the new Spot Perpetual Arbitrage strategy! This strategy lets you arbitrage between a spot and futures exchange. The differences between the prices on the spot and futures markets before the settlement date give a good sense of the direction in which both markets would go. Prices on both futures and spot will converge as the settlement date or time approaches.
+
+The idea of this strategy is to open a position in one direction and execute an order in the opposite direction on the spot. Learn more about it here #[2977](https://github.com/CoinAlpha/hummingbot/pull/2977).
```
Comment (dennisocana) [inline content/release-notes/0.37.0.mdx:40]: "~~Learn more about it here #[2977](https://github.com/CoinAlpha/hummingbot/pull/2977)~~" we should avoid directing users to PRs and direct them instead to the section in the docs where these changes are explained in more detail.

File: content/release-notes/0.37.0.mdx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+Read more about how to use the Beaxy connector [here](/exchange-connectors/Beaxy).
+
+## New Protocol Connector: Perpetual Finance
+
+In v0.37, Hummingbot has added a new protocol connector, [Perpetual Finance](https://perp.fi/).
+
+Perpetual protocol is a decentralized protocol to trade perpetual contracts for every asset powered by a Virtual Automated Market Maker (vAMM). The project aims to create novel financial instruments by democratizing Futures and other crypto-asset derivatives.
+
+Read more about how to use the Perpetual Finance connector [here](/protocol-connectors/perp-fi)
```
Comment (dennisocana) [inline content/release-notes/0.37.0.mdx:34]:
```suggestion
Read more about how to use the Perpetual Finance connector [here](/protocol-connectors/perp-fi).
```

For consistency.

File: content/release-notes/0.37.0.mdx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+## Introducing the new Spot Perpetual Arbitrage strategy (Beta)
+
+In v0.37, we introduce the new Spot Perpetual Arbitrage strategy! This strategy lets you arbitrage between a spot and futures exchange. The differences between the prices on the spot and futures markets before the settlement date give a good sense of the direction in which both markets would go. Prices on both futures and spot will converge as the settlement date or time approaches.
+
+The idea of this strategy is to open a position in one direction and execute an order in the opposite direction on the spot. Learn more about it here #[2977](https://github.com/CoinAlpha/hummingbot/pull/2977).
+
+## Improvement to Liquidity Mining strategy
```
Comment (dennisocana) [inline content/release-notes/0.37.0.mdx:42]: "Improvements" since there are more than one.

File: content/release-notes/0.37.0.mdx [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+In v0.36, we introduced the Liquidity Mining strategy. In v0.37, we made improvements to give users a better experience when using this strategy, we added two new parameters [`max_spread`](/strategies/liquidity-mining/#max_spread) and [`max_order_age`](/strategies/liquidity-mining/#max_order_age). Tables are also added to let you see more detail about your base and quote balance and also the campaign. Another improvement is better balance distribution on the selected `token`. Learn more about it here #[2961](https://github.com/CoinAlpha/hummingbot/pull/2961).
+
+## Bug Fixes
+
+- #[2949](https://github.com/CoinAlpha/hummingbot/pull/2949) Add `Exchanges Total` to `balance` command
+  - **We want to thank community member [Iamtha](https://github.com/lamtha) for these contribution!**
```
Comment (dennisocana) [inline content/release-notes/0.37.0.mdx:49]:
```suggestion
  - **We want to thank community member [Iamtha](https://github.com/lamtha) for this contribution!**
```

**Review by dennisocana — CHANGES_REQUESTED** [review]

**Comment by JeremyKono** [comment]
All requested changes have been addressed in the latest commit

File: content/release-notes/0.37.0.mdx (hunk 1 above)
Comment (dennisocana) [reply content/release-notes/0.37.0.mdx]: @JeremyKono this is not resolved 👆

Comment (JeremyKono) [reply content/release-notes/0.37.0.mdx]: Resolved in the latest commit

**Review by dennisocana — APPROVED** [review]

**Pull request merged**